packages/ui/src/App.tsx

```diff
@@ -1,32 +1,23 @@
 import './styles/App.css'
-import { CssBaseline, GlobalStyles } from '@mui/material'
+import { CssBaseline } from '@mui/material'
 import { AccountContextProvider } from './contexts/AccountsContext'
 import { ApiContextProvider } from './contexts/ApiContext'
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
 import { MultiProxyContextProvider } from './contexts/MultiProxyContext'
 import ToastContextProvider from './contexts/ToastContext'
 import { theme } from './styles/theme'
-import { css, ThemeProvider as MuiThemeProvider } from '@mui/material/styles'
+import { ThemeProvider as MuiThemeProvider } from '@mui/material/styles'
 import { AccountNamesContextProvider } from './contexts/AccountNamesContext'
 import { NetworkContextProvider } from './contexts/NetworkContext'
 import MainLayout from './components/layout/Main'
 import { WatchedAddressesContextProvider } from './contexts/WatchedAddressesContext'
-import { AutocompleteStyles } from './components/library/Autocomplete'
-
-const appGlobalStyles = css`
-  .MuiAutocomplete-popper {
-    margin-top: 0.75rem !important;
-    ${AutocompleteStyles}
-  }
-`
 
 const App = () => {
   const queryClient = new QueryClient()
 
   return (
     <MuiThemeProvider theme={theme}>
       <CssBaseline />
-      <GlobalStyles styles={appGlobalStyles} />
```

Comment:
As described in the comments without global styles for Autocomplete the feature disablePortal={false} will not work. It helps us to render big lists and it's related to the following discussion https://github.com/ChainSafe/Multix/pull/235#discussion_r1258931751

Reply:
Thanks for your help. I got rid of `disablePortal` altogether, keeping the standard way.